db/seeds.rb

```diff
@@ -37,5 +46,47 @@
   email: "jomo@example.com",
   password: "123456789", # high seructity!
   password_confirmation: "123456789",
-  role: Role.get(:superadmin)
-)
\ No newline at end of file
+  role: Role.get(:superadmin),
+  badge: Badge.get(:donor),
+  confirmed: true
+)
+User.create!(
```

Comment:
I'm 99% sure I removed them when moving into dev-new, but I will change it here too. I like having a seeded user as it makes testing fast and simple(er).